beacon-chain/usage.go

```diff
@@ -80,6 +80,8 @@ var appHelpFlagGroups = []flagGroup{
 	{
 		Name: "beacon-chain",
 		Flags: []cli.Flag{
+			cmd.MinimalConfigFlag,
+			cmd.E2EConfigFlag,
```

Comment:
this should be in `cmd`  instead of `beacon-chain`

Reply:
Can you elaborate on what you mean by this? Do you mean adding an exported array of per-system flags like in `featureconfig`?

Reply:
no I mean the namespace is incorrect for flag group. This is `beacon-chain` over here, if you go above this it has the `cmd` namespace, which is where this flag should be defined now.

Reply:
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/usage.go#L45

Reply:
Ah, I just figured the flags were more of a "beacon-chain" flag since it was related to config, I'll change it.